apps/condo/domains/common/hooks/useMultipleFiltersModal.tsx

```
@@ -490,7 +489,8 @@ const Modal: React.FC<MultipleFiltersModalProps> = ({
             setSelectedFiltersTemplate(updatedFilter || null)
         }
 
-        await handleSaveRef.current()
+        if (filtersValue.hasOwnProperty(NON_FIELD_ERROR_NAME)) delete filtersValue[NON_FIELD_ERROR_NAME]
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)